### PR TITLE
Domains multi: Wrap Free in a translate call

### PR DESCRIPTION
**Pull request by lsl**
## Proposed Changes

* Translation fix for "Free" in domains multi select flow

## Testing Instructions

http://calypso.localhost:3000/start/domains?flags=domains/add-multiple-domains-to-cart

### CODE REVIEW

File: client/signup/steps/domains/index.jsx [hunk 1]
```diff
@@ -814,7 +814,7 @@ export class RenderDomainsStep extends Component {
 										<BoldTLD domain={ this.state.wpcomSubdomainSelected.domain_name } />
 									</div>
 									<div className="domain-product-price__price">
-										<span className="domains__price-free">Free</span>
+										<span className="domains__price-free">{ this.props.tranlsate( 'Free' ) }</span>
```
Comment (candy02058912) [inline client/signup/steps/domains/index.jsx]:
```suggestion
										<span className="domains__price-free">{ this.props.translate( 'Free' ) }</span>
```

**Review by candy02058912 — CHANGES_REQUESTED** [review]
typo

**Review by candy02058912 — APPROVED** [review]
Tested LGTM 👍

**Comment by lsl** [comment]
Thanks for the fix, I tested this in a separate PR but decided that PR won't merge but this fix was still needed so created this PR separately.. without testing :disappointed:

**Comment by candy02058912** [comment]
> Thanks for the fix, I tested this in a separate PR but decided that PR won't merge but this fix was still needed so created this PR separately.. without testing 😞

No worries :super-teamwork: (they need to add this 🤣 )

**Pull request merged**